global/scripts/debug.shlib

```diff
@@ -0,0 +1,261 @@
+# Description:
+#
+#   Function for printing debugging information on caught errors.
+#   It makes use of error trapping. To reset the trap, add
+#   end of the script using "trap - ERR".
+#
+# Input:
+#
+#   — line number
```

Comment:
This dash appears to be unicode, I'd prefer if scripts were strict ascii.

Reply:
Changed all unicode dashes to regular ascii.